woocommerce/woocommerce
# Run all tests on daily run

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Currently, our daily test run only executes 5 of the 261 tests in the API test suite.  This PR changes that to execute the entire suite.

There are some test that fail when they aren't run agains `wp-env`.  For the time being, we'll skip those tests and work to fix their execution on CI.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1.  Go to https://github.com/woocommerce/woocommerce/actions/workflows/smoke-test-daily.yml
2. Under "Run workflow", select this branch (instead of trunk)
3. Run the workflow
4. Verify in the results that all API tests were run (261) and that all tests pass/are skipped (results should look like this run: https://github.com/woocommerce/woocommerce/actions/runs/5616661454/job/15219866462).



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
